proposals/wg-reference-spec.md

```diff
@@ -0,0 +1,82 @@
+# OCI Working Group: Reference Spec
+
+## Reference Spec OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Reference Spec Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+References are a string that is used by runtimes and other OCI registry clients to retrieve a container image.
+They are currently a convention, used by many clients, adopted from Docker and implemented in distribution/distribution.
+This WG seeks to define the syntax and parsing of a reference as an OCI spec.
```

Comment:
"define" is a loaded word for me here, given the pre-existing nature of the relevant format -- would prefer a word like "document"

Also, "in the OCI" instead of "as an OCI spec" (because as I've noted prior, I don't think this should be new a spec, but declaring that it _will_ be a new spec right off the bat in the WG proposal is definitely premature).

```suggestion
This WG seeks to document the syntax and parsing of a reference within the OCI.
```

or

```suggestion
This WG seeks to spec the syntax and parsing of a reference within the OCI.
```